cmake/CheckCompilerFunctionality/CheckStdChrono.cu

```diff
@@ -0,0 +1,7 @@
+// Including chrono is enough to trigger the compilation error being tested for. See:
+// https://github.com/FLAMEGPU/FLAMEGPU2/issues/575
+// https://gcc.gnu.org/bugzilla/show_bug.cgi?id=100102
+#include <chrono>
```

Comment:
These are new/fancy.

Reply:
They've been on the main repo for a while, but hadn't been duplciated here to detect known bad compiler configuraitons. Probably not really needed here, but no harm either.